src/modules/sih/sih.hpp

```diff
@@ -98,18 +96,12 @@ class Sih : public ModuleBase<Sih>, public ModuleParams
 	void parameters_update_poll();
 	void parameters_updated();
 
-	// to publish the sensor baro
-	struct sensor_baro_s            _sensor_baro {};
-	orb_advert_t                    _sensor_baro_pub{nullptr};
-	// to publish the sensor mag
-	struct sensor_mag_s             _sensor_mag {};
-	orb_advert_t                    _sensor_mag_pub{nullptr};
-	// to publish the sensor gyroscope
-	struct sensor_gyro_s            _sensor_gyro {};
-	orb_advert_t                    _sensor_gyro_pub{nullptr};
-	// to publish the sensor accelerometer
-	struct sensor_accel_s           _sensor_accel {};
-	orb_advert_t                    _sensor_accel_pub{nullptr};
+	// simulated sensor instances
+	PX4Accelerometer _px4_accel{ 1311244, ORB_PRIO_DEFAULT, ROTATION_NONE }; // 1311244: DRV_ACC_DEVTYPE_ACCELSIM, BUS: 1, ADDR: 1, TYPE: SIMULATION
```

Comment:
Can you use a macro/method for the ID?

Reply:
We could build them up piece by piece with multiple calls setting each thing, but it didn't seem worth it given we ultimately still have magic numbers floating around.

https://github.com/PX4/Firmware/blob/d2c824c5346f27907043efae6b81e3bf73ec51c8/ROMFS/px4fmu_common/init.d-posix/rcS#L99

Reply:
> We could build them up piece by piece with multiple calls setting each thing

I still prefer that.

Reply:
I'll come back to it in another pass system wide.